main.go

```diff
@@ -569,6 +571,7 @@ Do you really want to delete?
 		},
 	}
 
+	ssm.Run()
```

Comment:
You've brought me a very good idea making environment variables as the interface between helmfile and kv-backend like ssm. Good job.

Reply:
I think we can merge this as an experimental feature and start implementing more like this once you wrap behind a feature flag. Perhaps something like `if _, exists := os.LookupEnv("SSM_PATH"); exists { ssm.Run() }`?

Reply:
I agree this would be nice to have the `ssm.Run` not triggerd everytime helmfile is launched and you don't care about ssm at all.

Reply:
@sgandon @mumoshu look at the second line of the ssm.Run() function. It does that check already. I just didn’t want to make the main function messy with a bunch of checks.

Reply:
But also, I don’t mind just working on that more flexible solution proposed above. Like, we can keep this PR open (but on hold) just in case I hit a snag with that code implementation. 

I really just wanted to throw something together here to start the conversation so we could structure something awesome together. ☺️